configFiles/kubernetes/auctionDataManager.yaml

```diff
@@ -70,10 +70,6 @@ spec:
             - /loadData.pl
           initialDelaySeconds: 15
           periodSeconds: 10
-        resources:
-          requests:
-            cpu: 1
-            memory: 8Gi
```

Comment:
Will this change impact db load time?

Reply:
No.  The requests are only used for admission control.  For micro the 1 CPU request is more than the CPU requests of the non-dataServices pods, and so I was running into situations where it couldn't start the data manager pod even though there would be enough space for all of the instances once the data was prepared.